apps/portals-base/src/components/Footer.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import footerConfig from './config/footerConfig'
-import logoFooterConfig from './config/logoFooterConfig'
-import { ReactComponent as PoweredBySvg } from './portal-assets/poweredbysynapse.svg'
-import Versions from './portal-components/Versions'
+import { ReactComponent as PoweredBySvg } from '../portal-assets/poweredbysynapse.svg'
+import Versions from '../components/Versions'
 import { SynapseConstants, ExperimentalMode } from 'synapse-react-client'
+import { usePortalContext } from './PortalContext'
 
 function Footer() {
+  const { footerConfig, logoFooterConfig } = usePortalContext()
```

Comment:
Footer component now pulls its config from context passed by portal app instead of importing from a portal config folder.